libsystemd-journal.cabal

```diff
@@ -1,5 +1,5 @@
 name:                libsystemd-journal
-version:             1.4.4
+version:             1.5.0
```

Comment:
I don't think this is a mjor bump. I'll change this to `1.4.5`.

Reply:
:+1:  yes, fine. I just wanted to discriminate between the two versions.